src/Compilers/CSharp/Portable/Symbols/Symbol_Attributes.cs

```diff
@@ -353,14 +353,7 @@ internal virtual void PostDecodeWellKnownAttributes(ImmutableArray<CSharpAttribu
             if (lazyCustomAttributesBag.SetAttributes(boundAttributes))
             {
                 this.RecordPresenceOfBadAttributes(boundAttributes);
-                if (addToDiagnostics == null)
-                {
-                    this.AddDeclarationDiagnostics(diagnostics);
-                }
-                else
-                {
-                    addToDiagnostics.AddRange(diagnostics);
-                }
+                declarationDiagnostics.AddRange(diagnostics);
```

Comment:
This code is inside Symbol. I am assuming it is called of off symbol, which is the target of the attribute.